man/systemd-cat.xml

```diff
@@ -62,8 +62,9 @@
         <term><option>--identifier=</option></term>
 
         <listitem><para>Specify a short string that is used to
-        identify the logging tool. If not specified, no identification
-        string is written to the journal.</para></listitem>
+        identify the logging tool. If not specified, identification
+        string is the executable name, or <literal>cat</literal> if the
+        input is read from a pipe.</para></listitem>
```

Comment:
Not your fault, but that's slightly misleading.

`--identifier=` specifies SYSLOG_IDENTIFIER field of the journal entry, and if it is unset, the field actually will not set. However, each journal entry also has _COMM field, and it is automatically picked by journald from the PID of the process, hence _COMM will be `cat` or the name of the executed binary in this case.
journalctl will use SYSLOG_IDENTIFIER if it exists, and falls back to _COMM. So, when `--identifier=` is unspecified, people will see 'cat' or the executed binary name in the output of `journalctl`.

You can confirm that behavior with `journalctl --reverse --output=export`.